phs.ttl

```diff
@@ -190,6 +219,23 @@
                       skos:prefLabel "Diagnostic Method Type"@en .
 
 
+###  http://linked.data.gov.au/def/phs#Bycatch
+:Bycatch rdf:type owl:Class ;
+      rdfs:subClassOf :Stratum ,
+                      dwc:Taxon ;
+      owl:disjointWith :Substrate ;
+      rdfs:comment "An organism that was included in a sample, but was not the target of the surveillance activity. A 'bycatch' specimen may also be a Pest, targeted by other surveillance activities." ;
+      skos:example "A beetle that was found in a fruit fly trap." ;
+      skos:prefLabel "Bycatch" .
+
+
+###  http://linked.data.gov.au/def/phs#BycatchType
+:BycatchType rdf:type owl:Class ;
+          rdfs:subClassOf skos:Concept ;
+          skos:definition "The subtype of a Bycatch specimen"@en ;
+          skos:prefLabel "Bycatch Type"@en .
```

Comment:
@F3RR3T thinks that bycatch type may be unnecessary. review use case

Reply:
Yes, Bycatch is moot. We now record `:TaxonOccurrence` regardless of its pest (or Bycatch) status.